Magnetometer.h

```diff
@@ -0,0 +1,55 @@
+#include <Wire.h>
+#include <LIS3MDL.h>
+#include <math.h>
+
+struct Magnetic {
+	int x;
+	int y;
+	int z;
+};
+LIS3MDL mag;
+float magnetometer(float magnetic[]) {
+	mag.read();
+	delay(1);
+	Magnetic offset;
+	
+	offset.x = -2373 + 2700 - 1500 - 200; //XDDDDDDDDDDDDD
+	offset.y = 5909 - 1500 - 1500; //XDDDDDDDDDD
```

Comment:
Nie jasne czym są te liczby uzyj stałej ze znaczącą nazwą
const int NAZWA = wartosc;
#define NAZWA wartosc;

Reply:
to offsety, ale jak już pisałem one są do dokładnego policzenia przez nas (ta cała procedura z wyznaczaniem środka okręgu)